### PR TITLE
Fix part of #1028: Add Walkthrough app tests to the non-flaky queue 

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR adds tests in the Walkthrough directories of the app module to the non-flaky queue
Fixes part of #1028

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -115,8 +115,8 @@ jobs:
           name: Domain tests
           command: ./.circleci/gradle/gradlew -i :domain:test
       - run:
-          name: App -  FAQ, Help, Splash, Parser, RecyclerView, Utility tests
-          command: ./.circleci/gradle/gradlew -i :app:testDebugUnitTest --tests org.oppia.app.faq* --tests org.oppia.app.help* --tests org.oppia.app.splash* --tests org.oppia.app.parser* --tests org.oppia.app.recyclerview* --tests org.oppia.app.utility* 
+          name: App -  Walkthrough, FAQ, Help, Splash, Parser, RecyclerView, Utility tests
```
Comment (vinitamurthi) [inline .circleci/config.yml]: Can you write this alphabetically? i.e. put Walkthrough at the end

File: .circleci/config.yml [hunk 2]
```diff
@@ -115,8 +115,8 @@ jobs:
...
+          command: ./.circleci/gradle/gradlew -i :app:testDebugUnitTest --tests org.oppia.app.walkthrough* --tests org.oppia.app.faq* --tests org.oppia.app.help* --tests org.oppia.app.splash* --tests org.oppia.app.parser* --tests org.oppia.app.recyclerview* --tests org.oppia.app.utility*
```
Comment (vinitamurthi) [inline .circleci/config.yml]: Same here, add walkthrough at the end

**Review by vinitamurthi — APPROVED** [review]
LGTM! Just a minor ordering change. Feel free to submit this once the ordering is changed!

File: .circleci/config.yml (hunk 1 above)
Comment (veena14cs) [reply .circleci/config.yml]: Done.

File: .circleci/config.yml (hunk 2 above)
Comment (veena14cs) [reply .circleci/config.yml]: Done.

**Comment by vinitamurthi** [comment]
Can you once pull from develop? To ensure that you are not running the flaky tests queue as well in the PR

**Comment by veena14cs** [comment]
> Can you once pull from develop? To ensure that you are not running the flaky tests queue as well in the PR

Yes sure. Also please mark comments resolved if the changes looks fine.

**Comment by vinitamurthi** [comment]
Marked them as resolved!

**Comment by veena14cs** [comment]
> Marked them as resolved!

Thanks.

**Pull request merged**